### PR TITLE
fix: showcase build fail on initial setup

### PR DESCRIPTION
**Pull request by fpaul-1A**
## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

File: packages/@o3r/core/project.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
         "script": "postbuild"
       },
       "dependsOn": [
+        "^build",
```
Comment (kpanot) [inline packages/@o3r/core/project.json:17]: It is supposed to be the job of the `compile` task

File: apps/showcase/project.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
         "script": "postbuild:patch"
       },
       "dependsOn": [
+        "^build",
```
Comment (kpanot) [inline apps/showcase/project.json:33]: Not needed here, it is part of the `build-source`.
I also think that the `build-source` default dependency from `nx.json` is already including the `^build`

File: packages/@o3r/core/project.json (hunk 1 above)
Comment (fpaul-1A) [reply packages/@o3r/core/project.json]: indeed, this was unnecessary

File: nx.json [hunk 3]
```diff
@@ -187,6 +187,9 @@
       ]
     },
     "build-source": {
```
Comment (kpanot) [inline nx.json:189]: Maybe simpler to rename the `build-source` from the `showcase` to `compile` to be align with the other packages

**Review by kpanot — APPROVED** [review]

File: nx.json (hunk 3 above)
Comment (fpaul-1A) [reply nx.json:189]: I'd say it might be better to keep it as close as possible as the `angular.json` generated from new apps

**Review by mrednic-1A — APPROVED** [review]

**Pull request merged**